OTAcceleratorCore/OTMultiPartyCommunicator.h

```diff
@@ -169,4 +169,19 @@ typedef void (^OTMultiPartyCommunicatorBlock)(OTCommunicationSignal signal, OTMu
  */
 @property (readonly, nonatomic) OTStreamVideoType videoType;
 
+/**
+ *  An integer value to indicate the total connections.
+ */
+@property (readonly, nonatomic) NSUInteger connectionCount;
+
+/**
+ *  A boolean value to indicate if the connection is the oldest.
+ */
+@property (readonly, nonatomic) BOOL isFirstConnection;
```

Comment:
We had some requests about that in Android, so IMHO, it could be interesting to have in iOS as well